src/components/BlogPage/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface IBlog {
+  author: string;
+  content?: string;
```

Comment:
should content and summary be optional?

Reply:
so was meant to get your ops, basically i re use that type one for the card and one for the page. but the difference is one uses the summary and one uses the content

Reply:
hmmm, i would have a card type and then the blog type extends that and adds the extra content

surely both use summary if its stored in the same object on firestore?